microsoft/testsuites/core/kernel_modules.py

```diff
@@ -0,0 +1,70 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from typing import Dict, List
+
+from assertpy import assert_that
+
+from lisa import (
+    Environment,
+    Logger,
+    Node,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+    simple_requirement,
+)
+from lisa.sut_orchestrator import AZURE, HYPERV, READY
+from lisa.tools import KernelConfig
+
+
+@TestSuiteMetadata(
+    area="core",
+    category="functional",
+    description="""
+    This test suite covers validation of essential non hv kernel modules.
+    """,
+    requirement=simple_requirement(supported_platform_type=[AZURE, HYPERV, READY]),
+)
+class KernelModule(TestSuite):
+    @TestCaseMetadata(
+        description="""
+        This test case will
+        1. Verify the presence of essential kernel modules.
+        """,
+        priority=1,
+    )
+    def verify_essential_kernel_modules(
```

Comment:
It looks like an Azure specified test case, please move to azure standard test suite.